src/app/features/genes/components/gene-comparison-tool/components/gene-comparison-tool-details-panel/gene-comparison-tool-details-panel.component.html

```diff
@@ -18,10 +18,8 @@
         <div>P-Value</div>
       </div>
       <div>
-        <div>{{ getSignificantFigures(data.value, 3) }}</div>
-        <div>
-          {{ getSignificantFigures(data.pValue, 3) }}
-        </div>
+        <div [innerHTML]="getSignificantFigures(data.value, 3)"></div>
```

Comment:
What do you think about changing `getSignificantFigures` to return a character that will be displayed appropriately without using `[innerHTML]`, since that method is used across Agora so we may run into this issue again in another component? I think using the unicode symbol for em dash (`\u2014`) may work, but I haven't tested it!

Reply:
That's a great idea.  I wasn't super happy with having to specify the innerHTML so I think that's a more elegant approach!